js_modules/dagster-ui/packages/ui-core/src/runs/filterLogs.tsx

```diff
@@ -38,7 +40,7 @@ export function filterLogs(logs: LogsProviderLogs, filter: LogFilter, filterStep
             if (f.token === 'type') {
               return node.eventType && f.value === eventTypeToDisplayType(node.eventType);
             }
-            return node.message.toLowerCase().includes(f.value.toLowerCase());
+            return nodeTexts.some((text) => text.toLowerCase().includes(f.value.toLowerCase()));
```

Comment:
Could do `f.value.toLowerCase()` prior to the `some` call so that it's not repeated.